web/src/add_stream_options_popover.ts

```diff
@@ -31,7 +31,8 @@ export function initialize(): void {
             instance.setContent(parse_html(render_left_sidebar_stream_setting_popover()));
             popover_menus.on_show_prep(instance);
 
-            $("#streams_header").addClass("showing-streams-popover");
+            // TODO(evy) handle `showing-streams-popover` for tooltips per header
+            // $("#streams_header").addClass("showing-streams-popover");
```

Comment:
oh I think we don't need this because the popover is only on the search bar, but maybe make sure that looks okay